client/components/map/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { useState } from "react";
+export const useForceUpdate = () => {

```

Comment:
what's this for?

Reply:
It is used to update the state after changing the address, as the map was not changing when I entered a new address. This function exists in the MapPage itself so I will be removing it, but not sure if there is a better implementation for this